script.js

```diff
@@ -152,4 +152,17 @@ document.addEventListener('DOMContentLoaded', function() {
       this.setAttribute('aria-expanded', !isExpanded);
     });
   });
+
+  // If a section has more than 6 subsections, we collapse the list, and show a trigger to display them all
+  const seeAllTrigger = document.querySelector("#see-all-sections-trigger");
+  const subsectionsList = document.querySelector(".section-list");
+
+  if (subsectionsList && subsectionsList.children.length > 6) {
+    seeAllTrigger.setAttribute("aria-hidden", false);
+
+    seeAllTrigger.addEventListener("click", function(e) {
+      subsectionsList.classList.remove("section-list--collapsed");
+      seeAllTrigger.remove();
```

Comment:
`remove` is not supported in `ie11` which we still support :(
Could we do `seeAllTrigger.parentNode.removeChild(seeAllTrigger);` instead just to be on the safe side?

Reply:
Definitely! Changed in https://github.com/zendesk/copenhagen_theme/pull/70/commits/5b44b7d60be3e9f527687b04a9bec822e6589d02